staging/src/k8s.io/kube-scheduler/config/v1alpha2/types.go

```diff
@@ -77,6 +77,8 @@ type KubeSchedulerConfiguration struct {
 	// Duration to wait for a binding operation to complete before timing out
 	// Value must be non-negative integer. The value zero indicates no waiting.
 	// If this value is nil, the default value will be used.
+	// DEPRECATED: BindTimeoutSeconds in deprecated. To change volume
```

Comment:
please create an issue to track this and add a todo to the issue.

Reply:
https://github.com/kubernetes/kubernetes/issues/90958